nifi-nar-bundles/nifi-gcp-bundle/nifi-gcp-processors/src/main/java/org/apache/nifi/processors/gcp/bigquery/BigQueryUtils.java

```diff
@@ -37,29 +38,51 @@
     private final static Type gsonSchemaType = new TypeToken<List<Map>>() { }.getType();
 
     public static Field mapToField(Map fMap) {
-        String typeStr = fMap.get("type").toString();
+        String typeStr = fMap.get("type").toString().toUpperCase();
         String nameStr = fMap.get("name").toString();
-        String modeStr = fMap.get("mode").toString();
+        String modeStr;
+        if(fMap.containsKey("mode")) {
+        	modeStr = fMap.get("mode").toString();
+        } else {
+        	modeStr = Mode.NULLABLE.name();
+        }
         LegacySQLTypeName type = null;
```

Comment:
As we have a default case now, the null initialisation is not needed anymore.

Reply:
I don't understand. Whcih default case are you talking about ?

Reply:
I meant the default branch in the switch statement you added (in place of the former if-else construct, that had no else branch).